### ISSUE TITLE
support for RGB, HLS?

### ISSUE THREAD

**Issue by beckend** [doowb/ansi-colors#39]
when one needs more than the included.

**Comment by dimaslanjaka** [comment]
need also for HEX

**Comment by danneu** [comment]
Yeah, the readme says it's a drop-in replacement for `chalk`, but it's missing these methods that chalk has. The only reason I was using chalk was for `chalk.hex('#111111')` and `chalk.rgb(r,g,b)`.
